<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QEastMoneyHSGTDialog</class>
 <widget class="QDialog" name="QEastMoneyHSGTDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>市场</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="MarketTypeCBX">
       <item>
        <property name="text">
         <string>沪深</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>沪市</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>深市</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="DateEdit">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="minimumTime">
        <time>
         <hour>12</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::YearSection</enum>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2017</year>
         <month>7</month>
         <day>19</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NameEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="SearchBTB">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>64</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>代码</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>收盘价</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>涨跌幅</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>均价</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>总金额</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>外资买入</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>外资卖出</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>外资净额</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>持有量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>市值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>外资成本</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
